### ISSUE TITLE
Classes in models/concerns should not belong there

### ISSUE THREAD

**Issue by olivierobert** [shivabhusal/lacerate#18]
All classes / modules In `models/concerns` do not qualify as concerns. Some of them should be placed either in the `models` root directory e.g. `ResultNullObject` and `Query`. `GoogleScraper` would either fit in `lib` maybe in `models` (but that would be a bit strange). 

Since Rails is based on conventions, organizing files in the right subdirectories is important.

**Comment by shivabhusal** [comment]
Yes, I have put them in `model/concerns` so that I could just cut/paste them to somewhere more appropriate. But, my bad; I was in rush and forgot to re-arrange them :)
